webui/react/src/pages/Wait.tsx

```diff
@@ -41,6 +41,10 @@ const Wait: React.FC = () => {
     message = 'Missing required parameters.';
   } else if (waitStatus && terminalCommandStates.has(waitStatus.state)) {
     message = `${capitalizedTaskType} has been terminated.`;
+  } else if (capitalizedTaskType === 'Tensor-board'
+  && waitStatus
+  && waitStatus?.state === CommandState.Waiting) {
+    message = `Waiting for ${capitalizedTaskType} metrics step to be completed.`;
```

Comment:
So, technically we could be waiting for one of the following:

1. The `TensorboardFetcher` hasn't downloaded any files, which occurs when we haven't completed a step which produces metrics.
2. The `tensorboard` process is, for some reason, not starting or is otherwise producing a non-200 HTTP response.

99.99% of the time, it is going to be option 1. However, I wonder if we should change the verbiage here to say, "Waiting for tensorboard process to start. This delay will occur if we have not completed a metrics step." - Or some such thing?

I don't know though :shrug: